app/models/game.rb

```
@@ -54,75 +54,26 @@ def unique_game_per_teams_and_date
   has_one :home_team_season, through: :home_team_game, source: :team_season
   has_one :away_team_season, through: :away_team_game, source: :team_season
   has_many :predictions, dependent: :destroy
+  has_one :game_odd, dependent: :destroy
+  has_many :bookmaker_odds, dependent: :destroy
+  has_many :bet_recommendations, dependent: :destroy
+  has_many :current_bet_recommendations, -> { where(current: true) }, class_name: 'BetRecommendation'
```

RuboCop (1.75.5)

[convention] Specify a `:dependent` option.

(Rails/HasManyOrHasOneDependent)